test/plugin_functional/README.md

```diff
@@ -29,4 +29,29 @@ test plugin), you can just run:
 
 ```
 yarn start --plugin-path=test/plugin_functional/plugins/<plugin_folder>
-```
\ No newline at end of file
+```
+
+If you wish to start Kibana with multiple test plugins, you can run:
+
+```
+yarn start --plugin-path=test/plugin_functional/plugins/<plugin_folder1> --plugin-path=test/plugin_functional/plugins/<plugin_folder2> ... 
+```
+
+If you wish to load up specific es archived data for your test, you can do so via:
```

Comment:
I wonder if this shouldn't rather go to the general function test docs, since it's not specific for the plugin tests, and just place a reference here, towards the other docs?

Reply:
Sure, I had to create a new PR for it though because I just inline edited this and wasn't sure how to pull it down and modify